halo2-base/src/gates/range.rs

```diff
@@ -222,12 +229,10 @@ pub trait RangeInstructions<F: ScalarField> {
         self.check_less_than(ctx, a, Constant(self.gate().get_field_element(b)), range_bits)
     }
 
-    /// Checks that `a` is in `[0, b)`.
-    /// * a: value to check
-    /// * b: upper bound
-    /// TODO: Add Description
+    /// Performs a range check that `a` has at most `bit_length(b)` and then constrains that `a` is less than `b`.
     ///
-    /// Does not require bit assumptions on `a, b` because we range check that `a` has at most `bit_length(b)` bits.
+    /// * a: [AssignedValue] value to check
+    /// * b: upper bound expressed as a [BigUint] value
```

Comment:
This could be good to run tests on for what range `b` is valid in. E.g. if `b >= F::modulus()` what happens?